### PR TITLE
feat(og image): more configurations

### PR DESCRIPTION
**Pull request by giladgd**
* Added configuration for the result image width
* Added configuration for the max characters per line

### CODE REVIEW

File: packages/vitepress-plugin-og-image/src/vitepress/index.ts [hunk 1]
```diff
@@ -256,6 +260,18 @@ export interface BuildEndGenerateOpenGraphImagesOptions {
...
+   * Width of the result image.
+   * Defaults to `1200`.
+   */
+  resultImageWidth?: number
+
+  /**
+   * Maximum characters per line.
+   * Defaults to `17`.
+   */
+  maxCharactersPerLine?: number
```
Comment (nekomeowww) [inline packages/vitepress-plugin-og-image/src/vitepress/index.ts:274]:
```suggestion
  /**
   * Maximum characters per line.
   * Defaults to `17`.
   *
   * @default 17
   */
  maxCharactersPerLine?: number
```

File: packages/vitepress-plugin-og-image/src/vitepress/index.ts [hunk 2]
```diff
@@ -256,6 +260,18 @@ export interface BuildEndGenerateOpenGraphImagesOptions {
    * and will fallback to a builtin template.
    */
   templateSvgPath?: string
+
+  /**
+   * Width of the result image.
+   * Defaults to `1200`.
+   */
+  resultImageWidth?: number
```
Comment (nekomeowww) [inline packages/vitepress-plugin-og-image/src/vitepress/index.ts:268]:
```suggestion
  /**
   * Width of the result image.
   * Defaults to `1200`.
   *
   * @default 1200
   */
  resultImageWidth?: number
```

File: packages/vitepress-plugin-og-image/src/vitepress/index.ts (hunk 1 above)
Comment (giladgd) [reply packages/vitepress-plugin-og-image/src/vitepress/index.ts]: Fixed

File: packages/vitepress-plugin-og-image/src/vitepress/index.ts (hunk 2 above)
Comment (giladgd) [reply packages/vitepress-plugin-og-image/src/vitepress/index.ts]: Fixed

**Pull request merged**